packages/terra-clinical-slide-group/src/SlideGroup.jsx

```diff
@@ -0,0 +1,60 @@
+import React, { PropTypes } from 'react';
+import classNames from 'classnames';
+import CSSTransitionGroup from 'react-transition-group/CSSTransitionGroup';
+
+import Slide from './Slide';
+
+import './SlideGroup.scss';
+
+const propTypes = {
+  /**
+   * The array of components for the group. Only the last component is visible. The others are hidden but still mounted.
```

Comment:
```jsx
<div>
  <Slide 1 />
  <Slide 2 />
  <Slide 3 />
</div>
```
If we want 3 to be the top component, with a bit of absolute positioning, everything works out. If we wanted 1 to be the top component, we'd need to play around with z-index and everything else. Throw animations into the mix and its just plain easier this way.

I could always reverse the array before rendering, but :man_shrugging: